valdr/valdr-message.d.ts

```diff
@@ -0,0 +1,74 @@
+﻿// Type definitions for valdr.js v1.1.5 (valdr-message)
+// Project: https://github.com/netceteragroup/valdr
+// Definitions by: Kai Ilbertz <https://github.com/ilbertz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace valdr.message {
+
+    interface IValdrMessage {
```

Comment:
https://www.typescriptlang.org/docs/handbook/writing-declaration-files.html

> Naming Conventions
In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.